packages/terra-date-picker/src/DatePicker.jsx

```diff
@@ -8,6 +8,10 @@ import './DatePicker.scss';
 
 const propTypes = {
   /**
+   * Custom attributes to apply to the date input.
+   */
+  attrs: PropTypes.object,
```

Comment:
My personal preference would be to make this prop name a little more descriptive. I spoke with Jake and Matt Butler a little bit about this and he thought something like inputAttributes might be more clear.

I don't necessarily think this prop name and the forms prop name need to match, but if there was a generic name that fit all cases it would be more consistent. `attrs` feels a little too generic but I don't have any good alternatives.